backend/__tests__/integration/test-environment.js

```diff
@@ -46,17 +51,19 @@ export default class TestEnvironment extends NodeEnvironment {
         UserModule,
         AuthorizationModule,
         PrismaModule,
+        ActiveProfilesModule,
+        TestDataModule,
       ],
       controllers: [AppController],
       providers: [AppService],
     }).compile();
 
     const app = moduleFixture.createNestApplication();
 
+    const reflector = app.get(Reflector);
     app.useGlobalPipes(new ValidationPipe());
-    app.useGlobalInterceptors(
-      new ClassSerializerInterceptor(app.get(Reflector))
-    );
+    app.useGlobalInterceptors(new ClassSerializerInterceptor(reflector));
+    app.useGlobalGuards(new JwtAuthenticationGuard(reflector));
```

Comment:
`app.useGlobalGuards(new JwtAuthenticationGuard(reflector));` enforces authentication on every endpoint that does not have the `@Public()` decorator